src/Libraries/Analysis/Analysis.csproj

```diff
@@ -110,15 +110,15 @@
   </Target>
   <Target Name="AfterBuild" Condition=" '$(OS)' != 'Unix' ">
     <!-- Get System.Drawing.dll -->
-    <GetReferenceAssemblyPaths TargetFrameworkMoniker=".NETFramework, Version=v2.0">
+    <GetReferenceAssemblyPaths TargetFrameworkMoniker=".NETFramework, Version=$(ResourceGeneration_FrameworkVersion)">
```

Comment:
v2.0??  This will be nice for updating in the future.

Reply:
Do we possibly want to make this same kind of change for "4.5" -> "4.7" and "4.0" -> "12.0"? Not needed now, but just a thought for the future.